setup.py

```diff
@@ -38,6 +38,9 @@
     if os.path.exists(pjoin(d, '__init__.py')):
         packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
 
+package_data = {'nb_conda_kernels': [pjoin('logos', 'python', '*'),
```

Comment:
`nb_conda_kernels` is not "installing" kernelspecs for each environment... you could potentially do that and put the logos there... instead this extension create the kernelspec on the fly, so we need to tell it which logo to use for each kernel/envs... For *actual* kernels, the mechanism is just reading the kernel folder where the logos live...